src/collapse/collapse.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, Input} from 'angular2/angular2';
+
+@Directive({
+  selector: '[ngb-collapse]',
```

Comment:
Hmm, I wonder why we would prefer attribute vs. element here. Somehow I was assuming:

```html
<ngb-collapse [collapsed]="condition">
   Content to collapse goes here...
</ngb-collapse>
```

but honestly I'm not sure _why_ I would lean towards this setup. Thoughts?
```